lib/plugins/paragon-webpack-plugin/ParagonWebpackPlugin.js

```diff
@@ -0,0 +1,156 @@
+const { Compilation, sources } = require('webpack');
+const dotenv = require('dotenv');
+const fs = require('fs');
+const path = require('path');
+const {
+  getParagonVersion,
+  getParagonThemeCss,
+} = require('../../../config/data/paragonUtils');
+const {
+  injectMetadataIntoDocument,
+  // getMfeRuntimeConfig,
+  getParagonStylesheetUrls,
+  injectParagonCoreStylesheets,
+  injectParagonThemeVariantStylesheets,
+} = require('./utils');
+const resolvePrivateEnvConfig = require('../../resolvePrivateEnvConfig');
+
+// Add process env vars. Currently used only for setting the
+// server port and the publicPath
+dotenv.config({
+  path: path.resolve(process.cwd(), '.env.development'),
```

Comment:
This makes that the build(`npm run build`) process uses env.development setttings instead of the env